qontract_utils/qontract_utils/aws_policy_validator.py

```
@@ -0,0 +1,663 @@
+            "ReplicateObject",
+            "ReplicateTags",
+            "RestoreObject",
+            "SubmitMultiRegionAccessPointRoutes",
+            "TagResource",
+            "UntagResource",
+            "UpdateAccessGrantsLocation",
+            "UpdateJobPriority",
+            "UpdateJobStatus",
+            "UpdateStorageLensGroup",
+            "WriteGetObjectResponse",
+            "*Object", # for backward compatibility with old wildcard policies
```

Ruff (0.15.15)

[error] Insert at least two spaces before an inline comment

Insert spaces

(E261)